css-sizing-3/Overview.bs

```diff
@@ -53,6 +53,8 @@ Module interactions</h3>
 	<p>This module extends the 'width', 'height', 'min-width', 'min-height', 'max-width', 'max-height', and 'column-width'
 	features defined in [[!CSS2]] chapter 10 and in [[!CSS3COL]]
 
+	<p>The definition of the 'box-sizing' property in module supersedes the one in [[CSS-UI-3]].
```

Comment:
I think I'd probably make this a note?

Reply:
I made all the requested changes except this one.

If it is a note, then css-sizing does not normatively supersede css-ui, and we have two specs both claiming to the the canonical version. That sounds less than ideal. Merging as is for expediency, but feel free to fix after the fact if you still disagree.